src/datatablerow/datatablerow.jl

```diff
@@ -4,10 +4,6 @@ immutable DataTableRow{T <: AbstractDataTable}
     row::Int
 end
 
-function Base.getindex(r::DataTableRow, idx::AbstractArray)
```

Comment:
No, I think Cameron deleted it just because I said untested code should be removed. I should have been more explicit and say that my remark only applied to code which was now unused because of this PR.